pom.xml

```diff
@@ -76,12 +76,12 @@
       <dependency>
         <groupId>io.vertx</groupId>
         <artifactId>vertx-sql-client</artifactId>
-        <version>${vertx.version}-FOLIO</version>
+        <version>3.9.4-FOLIO</version>
```

Comment:
I didn't see this mentioned in the PR description... is this a necessary part of this PR?  If so, can you please explain why?

Reply:
When I had a discussion with Adam and Julian they suggested to use the latest vertx version with major version 3 (3.9.6).
But dependencies with the FOLIO prefix for 3.9.6 don't exist. So we changed the vertx version property value to be 3.9.6 and left some dependencies with 3.9.4-FOLIO.

Reply:
In the future, please add this context to the PR description so that it's clear why the change is being made.  Without this information the PR reviewer is left wondering if the change was intentional, or a mistake.